alacritty/src/ipc.rs

```diff
@@ -62,6 +63,14 @@ pub fn spawn_ipc_socket(options: &Options, event_proxy: EventLoopProxy<Event>) -
                     let event = Event::new(EventType::CreateWindow(options), None);
                     let _ = event_proxy.send_event(event);
                 },
+                SocketMessage::Config(ipc_config) => {
+                    let window_id = ipc_config
+                        .window_id
+                        .filter(|id| id >= &0)
+                        .map(|id| WindowId::from(id as u64));
```

Comment:
That looks like `try_into` and `flat_map`.

Reply:
`flat_map` is not a method on `Option`.

Reply:
Oh, that's sad. Since you have `flatten` and `map` I thought that it had `flat_map`?

You can still do something with `try_into` and `ok`, but the way it's fine then.

Reply:
Switched to `and_then`.